test/specs/runApexTests.e2e.ts

```diff
@@ -0,0 +1,441 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { step } from 'mocha-steps';
+import {
+  CodeLens,
+  InputBox,
+  QuickOpenBox,
+  SideBarView,
+  TextEditor,
+  TreeItem
+} from 'wdio-vscode-service';
+import {
+  TestSetup
+} from '../testSetup';
+import * as utilities from '../utilities';
+
+describe('Run Apex Tests', async () => {
+  let prompt: QuickOpenBox | InputBox;
+  let testSetup: TestSetup;
+  const fiveMinutes = 5 * 60;
+
+  step('Set up the testing environment', async () => {
+    testSetup = new TestSetup('RunApexTests', false);
+    await testSetup.setUp();
+
+    // Create Apex class 1 and test
+    await utilities.createApexClassWithTest('ExampleApexClass1');
+
+    // Create Apex class 2 and test
+    await utilities.createApexClassWithTest('ExampleApexClass2');
+
+    // Create Apex class 3 and test
+    await utilities.createApexClassWithTest('ExampleApexClass3');
+
+    // Push source to scratch org
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Scratch Org and Override Conflicts', 6);
+  });
+
+  step('Run All Tests via Apex Class', async () => {
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+
+    // Open an existing apex test (e.g. BotTest.cls, search for @isTest)
```

Comment:
I'm not seeing anything to do with "search for @isTest".  Is this a stale comment and this should be removed?